src/lib/db/migrations/20230329093154_transaction-table-modify-cols.ts

```diff
@@ -0,0 +1,22 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table('transaction', function (table) {
+    table.dropColumn('token_id')
+    table.dropColumn('token_type')
+    table.enu('api_type', ['match2', 'order', 'capacity'], { useNative: true, enumName: 'api_type' })
+    table.enu('transaction_type', ['creation', 'proposal', 'accept'], { useNative: true, enumName: 'transaction_type' })
+  })
+}
+
+export async function down(knex: Knex): Promise<void> {
+  return knex.schema.table('transaction', function (table) {
+    table.integer('token_id')
+    table.enu('token_type', ['DEMAND', 'MATCH2'], {
```

Comment:
there's an issue in that `token_type` was `notNullable()` when originally created and the `down` isn't restoring that, so migrating up to this and down would delete the `token_type` value for existing transactions, but in the interest of moving on I'm going to approve